src/test/Simulation.t.sol

```diff
@@ -21,6 +21,7 @@ import "ds-test/test.sol";
 import "dss-interfaces/Interfaces.sol";
 import { UniswapV2CalleeDai } from "../UniswapV2Callee.sol";
 import { UniswapV2LpTokenCalleeDai } from "../UniswapV2LpTokenCallee.sol";
+import { UniV3Callee } from "../UniV3Callee.sol";
```

Comment:
There are a whole battery of tests for each of the other types: `UniswapV2Callee.t.sol`, `OasisDexCallee.t.sol`, are we missing a suite of tests at `UniV3Callee.t.sol`?

Reply:
Also `UniV3CalleeCharter.t.sol`

Reply:
`UniswapV2Callee.t.sol` and `OasisDexCallee.t.sol` hold legacy tests with no mainnet fork simulation and instead use a mockup exchange function. Hence the existence of `Simulation.t.sol`, which performs simulation tests for UniswapV2. In this PR I added two tests for V3: `testTakeLinkV3Profit` and `testTakeLinkV3NoProfit`. Please let me know if you consider there are things we should be testing which we are currently not.